helm/ako/crds/ako.vmware.com_sslprofiles.yaml

```diff
@@ -0,0 +1,134 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.17.2
+  name: sslprofiles.ako.vmware.com
+spec:
+  group: ako.vmware.com
+  names:
+    kind: SSLProfile
+    listKind: SSLProfileList
+    plural: sslprofiles
+    shortNames:
+    - sslprofile
+    singular: sslprofile
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: SSLProfile is the Schema for the ssl profile API
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SSLProfileSpec defines the desired state of SSLProfile
+            properties:
+              accepted_ciphers:
+                type: string
+              accepted_versions:
+                items:
+                  description: SSLVersionType represents an ssl version type
+                  enum:
+                  - SSL_VERSION_SSLV3
+                  - SSL_VERSION_TLS1
+                  - SSL_VERSION_TLS1_1
+                  - SSL_VERSION_TLS1_2
+                  - SSL_VERSION_TLS1_3
+                  type: string
+                type: array
+              cipher_enums:
```

Comment:
`accepted_cipher` and `cipher_enums` is oneOf the field. You can choose one of the two